src/main/scala/cromwell/engine/backend/runtimeattributes/LocalDisk.scala

```diff
@@ -0,0 +1,51 @@
+package cromwell.engine.backend.runtimeattributes
+
+import scala.util.Try
+import scalaz._
+import scalaz.Scalaz._
+import com.google.api.services.genomics.model.Disk
+
+object LocalDisk {
+  def parse(s: String): Try[LocalDisk] = {
+    val validation = s.split("\\s+") match {
+      case Array(name, DiskType.LOCAL.diskTypeName) =>
+        LocalDisk(name, DiskType.LOCAL).successNel[String]
+      case Array(name, sizeGb, diskType) if diskType != DiskType.LOCAL.diskTypeName =>
+        (validateLong(sizeGb) |@| validateDiskType(diskType)) { (s, dt) => LocalDisk(name, dt, s) }
+      case _ => s"'$s' should be in form 'NAME SIZE TYPE', with SIZE blank for LOCAL, otherwise SIZE in GB".failureNel
+    }
+
+    Try(validation match {
+      case Success(localDisk) => localDisk
+      case Failure(nels) => throw new UnsupportedOperationException(nels.list.mkString("\n"))
+    })
+  }
+
+  private def validateDiskType(diskTypeName: String): ValidationNel[String, DiskType] = {
+    DiskType.values().find(_.diskTypeName == diskTypeName) match {
+      case Some(diskType) => diskType.successNel[String]
+      case None =>
+        val diskTypeNames = DiskType.values.map(_.diskTypeName).mkString(", ")
+        s"Disk TYPE $diskTypeName should be one of $diskTypeNames".failureNel
+    }
+  }
+
+  private def validateLong(value: String): ValidationNel[String, Long] = {
```

Comment:
We don't have this validate method somewhere else already ? I think I've seen it before with a bunch of other ones for Boolean, Int etc... IMO this could even be something to put in lenthall.